addons/l10n_sa/__manifest__.py

```diff
@@ -2,18 +2,14 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 {
     'name': 'Saudi Arabia - Accounting',
-    'version': '2.0',
-    'author': 'DVIT.ME',
+    'version': '2.1',
+    'author': 'Odoo S.A., DVIT.ME',
     'category': 'Accounting/Localizations/Account Charts',
     'description': """
-Odoo Arabic localization for most arabic countries and Saudi Arabia.
-
-This initially includes chart of accounts of USA translated to Arabic.
-
-In future this module will include some payroll rules for ME .
+Odoo Arabic localization for most Saudi Arabia.
 """,
     'website': 'http://www.dvit.me',
-    'depends': ['account', 'l10n_multilang'],
+    'depends': ['l10n_multilang', 'l10n_gcc_invoice'],
```

Comment:
```suggestion
    'depends': [
        'l10n_multilang',
        'l10n_gcc_invoice',
    ],
```
Same for the PoS module